lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -940,6 +968,15 @@ public function addGroupBy($groupBy/*, string ...$groupBys*/)
             return $this;
         }
 
+        if (is_array($groupBy)) {
+            Deprecation::triggerIfCalledFromOutside(
```

Comment:
`QueryBuilder` isn't used by the DBAL, its API is only called from tests, should be `trigger()`.